cmd/kube-apiserver/app/server.go

```diff
@@ -396,8 +397,9 @@ func buildGenericConfig(
 			return
 		}
 	}
-
-	genericConfig.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(generatedopenapi.GetOpenAPIDefinitions, openapinamer.NewDefinitionNamer(legacyscheme.Scheme, extensionsapiserver.Scheme, aggregatorscheme.Scheme))
+	// wrap the definitions to revert any changes from disabled features
+	getOpenAPIDefinitions := openapi.GetOpenAPIDefinitionsWithoutDisabledFeatures(generatedopenapi.GetOpenAPIDefinitions)
+	genericConfig.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(getOpenAPIDefinitions, openapinamer.NewDefinitionNamer(legacyscheme.Scheme, extensionsapiserver.Scheme, aggregatorscheme.Scheme))
```

Comment:
does this mean CRD enums are stripped too, or just the ones of native APIs? What about aggregation?

Reply:
This only affect built-in, CRD schemas were untouched.

Reply:
Enums are removed from generated openapi defs of only built-ins. Because it is eliminated at the source, aggregation won't have enums for built-ins either.